src/Response/DTO/ParcelShop.php

```diff
@@ -31,16 +31,11 @@ public function __construct(
 
     public static function fromRawParcelShop(RawParcelShop $rawParcelShop): self
     {
-        $isCodHandler = in_array(FeatureEnum::AcceptsCash->value, $rawParcelShop->features, true);
-        $canPayByBankCard = in_array(FeatureEnum::AcceptsCard->value, $rawParcelShop->features, true);
-        $isParcelLocker = $rawParcelShop->type === ParcelShopTypeEnum::ParcelLocker->value;
         $contact = Address::fromResponse($rawParcelShop->contact);
```

Comment:
this one more could be removed

Reply:
This one is used in two places: [line 47](https://github.com/answear/gls-bundle/pull/7/files#diff-8e5a7e664b841584c3a4b31df4553b4c63d861cf0060a42f3fe0b9b96a94da27R47) and [line 53](https://github.com/answear/gls-bundle/pull/7/files#diff-8e5a7e664b841584c3a4b31df4553b4c63d861cf0060a42f3fe0b9b96a94da27R53)